utils/jsonutil/strings.go

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package jsonutil
+
+import (
+	"encoding/json"
+
+	"github.com/mysteriumnetwork/node/logconfig"
+)
+
+var log = logconfig.NewLogger()
+
+// ToJson marshals v to a JSON string
+func ToJson(v interface{}) string {
+	bytes, err := json.Marshal(&v)
+	log.Warnf("failed to marshal to JSON string: %v", err)
```

Comment:
We probably lacking if block here to check the `err`.

Reply:
This is intended. I'd like it to use in simple one-line situations, where error checking is not mandatory, e.g.:

```go
log.Info("user configuration loaded: \n", jsonutil.ToJson(cfg))
```

Reply:
But you are logging every time record `failed to marshal to JSON string` even if no error happened. Or I missing something?

Reply:
Ah I misunderstood you. Thanks, changed it. :)